src/Sulu/Bundle/SnippetBundle/Resources/public/js/components/snippet/form/details/main.js

```diff
@@ -157,6 +158,8 @@ define(['app-config'], function(AppConfig) {
                     this.initSortableBlock();
                     this.bindFormEvents();
 
+                    this.sandbox.emit('sulu.content.initialized', data);
```

Comment:
Then it actually has the wrong name... But lets keep it like that for the sake of consistency.